README.md

```diff
@@ -12,7 +12,7 @@ https://samtecspg.github.io/articulate/
 
 ## QUICK START
 
-1. Make sure you have Docker and `docker-compose` installed:
```

Comment:
@btotharye did you mean to remove step 1 and not replace it. I think we should leave the Quick start alone with a note at the top or bottom to check out the documentation for installations other than on localhost.

Reply:
sorry was accident should be fixed now

Reply:
do you want to update it from what I have now to include please see the documentation for more advanced setups or how do you want that to look?